google/async_resumable_media/requests/__init__.py

```diff
@@ -658,13 +658,13 @@ def SimpleUpload(*args, **kwargs):
    >>> json_response[u'name'] == blob_name
    True
 """
-from google.resumable_media.requests.download import ChunkedDownload
-from google.resumable_media.requests.download import Download
-from google.resumable_media.requests.upload import MultipartUpload
-from google.resumable_media.requests.download import RawChunkedDownload
-from google.resumable_media.requests.download import RawDownload
-from google.resumable_media.requests.upload import ResumableUpload
-from google.resumable_media.requests.upload import SimpleUpload
+from google.async_resumable_media.requests.download import ChunkedDownload
```

Comment:
Again, have you edited any of these constants? if not, de-duplicate.

Reply:
I think that we want to keep this since the class is modified